composer.json

```diff
@@ -18,21 +23,23 @@
     "guzzlehttp/guzzle": "^7.1",
     "symfony/serializer": "^5.1",
     "symfony/property-access": "^5.1",
-    "webmozart/assert": "^1.9",
-    "doctrine/annotations": "^1.11"
+    "webmozart/assert": "^1.9"
   },
   "require-dev": {
     "phpunit/phpunit": "^8.5.8",
     "squizlabs/php_codesniffer": "~3.5",
-    "friendsofphp/php-cs-fixer": "~2.18.3",
```

Comment:
Was this removed because it has a require for annotations as well?

Reply:
yessir seemingly